changelog/issue-6237.md

```diff
@@ -0,0 +1,5 @@
+audience: general
+level: patch
+reference: issue 6237
+---
+This patch fixes the case where a generic worker won't upload its log on a malformed payload error. This has been broken since v48.2.0 from PR [#6107](https://github.com/taskcluster/taskcluster/pull/6107).
```

Comment:
```suggestion
Fix the case where a generic worker won't upload its log on a malformed payload error. This has been broken since v48.2.0 from PR [#6107](https://github.com/taskcluster/taskcluster/pull/6107).
```

probably better to phrase it from the changelog reader point of view?

Reply:
Fix/Fixed not sure :)